app/models/user.rb

```diff
@@ -45,6 +45,11 @@ def updating_without_name_changes
 
   scope :members, -> { where(member: true) }
 
+  scope :membership_expires_in_x_days, -> (num_days){ includes(:payments).where("payments.payment_type = ? AND payments.expire_date = ?", Payment::PAYMENT_TYPE_MEMBER, (Date.current + num_days) ).order('payments.expire_date').references(:payments) }
+
+  scope :company_hbrand_expires_in_x_days, -> (num_days){ includes(:payments).where("payments.payment_type = ? AND payments.expire_date = ?", Payment::PAYMENT_TYPE_BRANDING, (Date.current + num_days) ).order('payments.expire_date').references(:payments) }
```

Comment:
We need to add, to both of the scopes, the filter `payments.payment_status = 'betald'`, since abandoned payments will remain in the DB with status == `'avvaktan'` (pending).

Reply:
Probably should add test that confirm we filter out such pending payment records.